Catappult/appcoins-wallet-android
# WAL-72 - Implemented Rating dialogs

**What does this PR do?**

   Implements rating dialog for wallet. Note that the correct API key for zendesk is not yet here!

**Database changed?**

   No

**How should this be manually tested?**

  Check ticket for conditions and verify that the dialog appears in those conditions (note that after discussing, the succesful transaction number doesn't have to be consecutive anymore). Also check that everything works with the dialog and it behaves as expected.

**What are the relevant tickets?**

  [WAL-72](https://aptoide.atlassian.net/browse/WAL-72)

**Code Review Checklist**
